supertokens_python/recipe/session/interfaces.py

```diff
@@ -589,20 +599,23 @@ def __init__(self, is_valid: bool, reason: Optional[Dict[str, Any]] = None):
 
 
 class SessionClaimValidator(ABC):
-    def __init__(self, id_: str):
+    def __init__(
+        self,
+        id_: str,
+        claim: SessionClaim[_T],  # pyright: ignore[reportInvalidTypeVarUse]
+    ) -> None:
         self.id = id_
-        self.claim: Optional[
-            SessionClaim[Any]
-        ] = None  # Child class must set this if required.
+        self.claim = claim
```

Comment:
On second thought, my previous comment about making `should_refetch` and `claim` compulsory was incorrect. What you should do is to change this back to make `claim` optional and `should_refetch` not be abstract. Then when you use these functions, you should check if `claim` is present, and if it is, only then call the `should_refetch` function, else not.

Reply:
Done.